### PR TITLE
Revert "Use the old VS2019 image to work around the new being broken"

### PR DESCRIPTION
**Pull request by johnor**
This reverts commit e96daac2ac49864ddfd105fce97b4a0d602dc884.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -4,17 +4,22 @@ configuration:
   - Debug
   - Release
 
+matrix:
+  allow_failures:
```
Comment (robinlinden) [inline appveyor.yml]: Delete this? :P

Comment (johnor) [reply appveyor.yml]: Looks like they are still failing unfortunately: https://ci.appveyor.com/project/robinlinden/n-e-s/builds/30045859

Comment (robinlinden) [reply appveyor.yml]: Ah, I assumed they fixed something. Are we sure we're not doing anything that causes UB or something?

Might be better to disable the jobs entirely for now if we're not doing anything stupid. The failing ones run for >1h and the jobs run in sequence on every push, so if you rebase a PR twice, that's 2h before your CI runs again. :/

Comment (robinlinden) [reply appveyor.yml]: There's also a Visual Studio 2019 Preview image we could try.

Comment (johnor) [reply appveyor.yml]: Ok, I can try that. But can we merge this now to start with so we at least can get new PR merged?

**Pull request merged**